documentation/book/api/index.adoc

```diff
@@ -2289,6 +2289,95 @@ endif::internal-generation[]
 === Topics
 
 
+[.createTopic]
+==== createTopic
+
+`POST /admin/topics`
+
+
+
+===== Description
+
+Creates a topic with given name, partitions count and replication factor.
+
+
+// markup not found, no include::{specDir}admin/topics/POST/spec.adoc[opts=optional]
+
+
+
+===== Parameters
+
+
+
+[cols="2,3,1,1,1"]
+.Body Parameter
+|===
+|Name| Description| Required| Default| Pattern
+
+| NewTopic
+| Creates a topic with given name, partitions count and replication factor. <<NewTopic>>
+| X
+| 
+| 
+
+|===
+
+
+
+
+
+===== Return Type
+
+
+
+
+-
+
+===== Content Type
+
+* application/vnd.kafka.v2+json
+
+===== Responses
+
+.HTTP Response Codes
+[cols="2,3,1"]
+|===
+| Code | Message | Datatype
+
+
+| 201
+| Created
+|  <<>>
+
+
+| 400
+| Request body is null or it doesn&#39;t contain topic name.
+|  <<Error>>
+
+|===
+
+===== Samples
+
+
+// markup not found, no include::{snippetDir}admin/topics/POST/http-request.adoc[opts=optional]
```

Comment:
because of how the openapi-generator works for building the doc you would need to add an `http-request.adoc` file with an example request in the `/documentation/book/api/snippet/admin/topics/POST` folder (you need to create the subfolders path `/admin/topics/POST`).
For the adoc file you could take the consumer creation as an example https://github.com/strimzi/strimzi-kafka-bridge/blob/main/documentation/book/api/snippet/consumers/%7Bgroupid%7D/POST/http-request.adoc
We just need an example JSON body.